functions/access.js

```diff
@@ -0,0 +1,142 @@
+const admin = require("firebase-admin");
+const functions = require("firebase-functions");
+const { HttpsError } = require("firebase-functions/lib/providers/https");
+const { passwordsMatch } = require("./auth");
+const { uuid } = require("uuidv4");
+
+const checkIsValidToken = async (venueId, uid, token) => {
+  console.log("asdasd");
+  if (!uid) return false;
+
+  console.log("asdasd");
+  const venueRef = admin.firestore().collection("venues").doc(venueId);
+  const accessRef = admin.firestore().collection("accessgranted").doc(uid);
+
+  try {
+    console.log("asdasd");
+    await admin.firestore().runTransaction(async (t) => {
```

Comment:
Aren't we supposed to wrap it in `try/catch` and handle if an error occurs?

Reply:
I believe I commented somewhere else in this review about it, but since this is a `Promise`, we can use it's `.catch()` directly, and `return false` from that or similar. So we don't need to use this 'external' `try`/`catch` around it.

Reply:
this appears to have been fixed now